akka-docs-dev/rst/java/stream-cookbook.rst

```diff
@@ -133,8 +133,8 @@ stream cannot continue without violating its resource bound, in this case
 By extracting the parts specific to *wordcount* into
 
 * a ``groupKey`` function that defines the groups
-* a ``foldZero`` that defines the zero element used by the fold on the substream given the group key
-* a ``fold`` function that does the actual reduction
+* a ``map`` map each element to value that is used by the reduce on the substream
+* a ``reduce`` function that does the actual reduction
```

Comment:
Is this change correct? A reduce needs at least two elements, while a wordcount is obviously doable with only one word of a kind.